packages/apps/plugins/plugin-stack/src/components/StackMain.tsx

```diff
@@ -146,41 +155,29 @@ const StackMain: FC<{ stack: StackType; separation?: boolean }> = ({ stack, sepa
         onOver={handleOver}
         onDeleteSection={handleDelete}
         onNavigateToSection={handleNavigate}
+        onAddSection={handleAddSection}
         collapsedSections={collapsedSections}
         onChangeCollapsedSections={onChangeCollapsedSections}
       />
 
-      <div role='none' className='flex justify-center mlb-4'>
+      <div role='none' className='flex justify-center mbs-4 pbe-4'>
         <ButtonGroup classNames={[surfaceElevation({ elevation: 'group' }), staticDefaultButtonColors]}>
-          <DropdownMenu.Root modal={false}>
-            <DropdownMenu.Trigger asChild>
-              <Button variant='ghost' data-testid='stack.createSection'>
-                <Plus className={getSize(6)} />
-              </Button>
-            </DropdownMenu.Trigger>
-            <DropdownMenu.Content>
-              <DropdownMenu.Arrow />
-              <DropdownMenu.Viewport>
-                {stackPlugin?.provides?.stack.creators?.map(({ id, testId, intent, icon, label }) => {
-                  const Icon = icon ?? Placeholder;
-                  return (
-                    <DropdownMenu.Item
-                      key={id}
-                      id={id}
-                      data-testid={testId}
-                      onClick={async () => {
-                        const { data: nextSection } = (await dispatch(intent)) ?? {};
-                        handleAdd(nextSection);
-                      }}
-                    >
-                      <Icon className={getSize(6)} />
-                      <span>{typeof label === 'string' ? label : t(...(label as [string, { ns: string }]))}</span>
-                    </DropdownMenu.Item>
-                  );
-                })}
-              </DropdownMenu.Viewport>
-            </DropdownMenu.Content>
-          </DropdownMenu.Root>
+          <Button
```

Comment:
do we think this is still needed since you can add before/after any section?

Reply:
It’s a good question… I mostly kept it as company for the empty state (consistency helps here I think) and for the file upload button (which seems exotic and weird on its own). I think we can consider adjusting the design all-up in a separate increment.